pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -260,3 +285,16 @@ func newCronJob() (*batchv1.CronJob, error) {
 
 	return obj.(*batchv1.CronJob), nil
 }
+
+func createDefaultEtcdBackupSpec() backupv1alpha1.EtcdBackupSpec {
```

Comment:
I don't think this belongs in this controller

Reply:
so this is the `noconfig` specs .. it is not final, we need to agree on them .. but this allow us to run the container without user applied config, and disable it when the use apply their own Backup CR